parm/atm/obs/testing/sondes.yaml

```diff
@@ -0,0 +1,106 @@
+obs space:
+  name: sondes
+  obsdatain:
+    engine: 
+      type: H5File
+      obsfile: sondes_obs_$(OBS_DATE).nc4
+    obsgrouping:
+      group variables: ["station_id"]
+      sort variable: "air_pressure"
+      sort order: "descending"
+  obsdataout:
+    engine:
+      type: H5File
+      obsfile: sondes_diag_$(OBS_DATE).nc4
+  simulated variables: [surface_pressure, air_temperature, eastward_wind, northward_wind, specific_humidity]
+geovals:
+  filename: sondes_geoval_$(OBS_DATE).nc4
+vector ref: GsiHofXBc
+tolerance: 0.01
+obs operator:
+  name: Composite
+  components:
+   - name: VertInterp
+     variables:
+     - name: air_temperature
+     - name: eastward_wind
+     - name: northward_wind
+     - name: specific_humidity
+   - name: SfcPCorrected
+     da_psfc_scheme: GSI
+     geovar_sfc_geomz: surface_geometric_height
+     geovar_geomz: geopotential_height
+     variables: 
+     - name: surface_pressure
+linear obs operator:
+  name: Composite
+  components:
+   - name: VertInterp
+     variables:
+     - name: air_temperature
+     - name: eastward_wind
+     - name: northward_wind
+     - name: specific_humidity
+   - name: Identity
+     variables:
+     - name: surface_pressure
+obs filters:
+  #
+  # Reject all obs with PreQC mark already set above 3
+  - filter: PreQC
+    maxvalue: 3.0
+    action:
+      name: reject
+  #
+  # Observation Range Sanity Check
+  - filter: Bounds Check
+    filter variables:
+    - name: surface_pressure
+    minvalue: 37499.0
+    maxvalue: 106999.0
+    action:
+      name: reject
+  #
+  # Assign obsError
+  - filter: Perform Action
+    filter variables:
+    - name: surface_pressure
```

Comment:
I see lots of snake style variable names in this file. Just to be clear, this is correct?

Reply:
for now, yes, we will have to go through GDASApp soon and change these once the code sprint work is in UFO/develop

Reply:
OK